src/ruby-github-release.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -e
+
+throw() { 
+  echo "$@" 1>&2
+  exit 1
+}
+
+# Ensure required env vars are set.
+[ -z "$GITHUB_TOKEN" ] && throw "GITHUB_TOKEN not set"
+# [ -z "$CIRCLE_PROJECT_REPONAME" ] && throw "CIRCLE_PROJECT_REPONAME not set"
+# [ -z "$CIRCLE_PROJECT_USERNAME" ] && throw "CIRCLE_PROJECT_USERNAME not set"
+
+# Ensure https://github.com/aktau/github-release is installed.
+command -v github-release > /dev/null 2>&1 || throw "Unable to locate github-release binary"
+
+# Ensure a single .gemspec file is present.
+GEMSPEC_COUNT=$(find . -type f -name "*.gemspec" | wc -l)
+[ "$GEMSPEC_COUNT" != "1" ] && throw "Expecting a single .gemspec file"
```

Comment:
At least on my system, the `wc` command returns tabs in front of the number, so this command will always fail. Might need to pipe this into `tr` to ensure no whitespace.

```bash
find . -type f -name "*.gemspec" | wc -l | tr -d [:space:]

Reply:
Interesting. This check works fine for me. I'll add the `tr` bit just to be safe tho

Reply:
Updated.